app/tutors/main.py

```diff
@@ -17,7 +18,7 @@
     dependencies=[MUBProtection],
     prefix="/mub/tutor-service",
 )
-
+mub_router.include_router(subjects_mub.router)
```

Comment:
polish: Пустые строчки существуют для читабельности
```suggestion
mub_router.include_router(subjects_mub.router)

```